packages/headless/src/api/commerce/unified-api/unified-api-params.ts

```diff
@@ -0,0 +1,88 @@
+import {URLPath} from '../../../utils/url-utils';
+import {pickNonBaseParams} from '../../api-client-utils';
+import {BaseParam} from '../../platform-service-params';
+
+export interface PlacementParam {
+  placementId: string;
+}
+
+export interface ImplementationParam {
+  implementationId?: string;
+}
+
+export interface ModeParam {
+  mode?: 'LIVE' | 'SAMPLE' | 'PREVIEW' | 'NONE';
```

Comment:
I think those need to be all upper case ? I vaguely remember getting an error back otherwise (ie: requesting `mode=live`)

Reply:
Confirmed, I got that error too